### DISCUSSION TITLE
Hot Reload (python)

### DISCUSSION THREAD

**Discussion by bchan9803** [flet-dev/flet#241] [Q&A]

**Comment by FeodorFitsner** [comment]
You just run your program with `flet <your-program.py>`, then don't close a window, change code, save and watch the contents of the window instantly refreshed.

**Reply by ECHARAOUI** [reply]
thank you bro

**Comment by MichaelDecent** [comment]
mine keeps hanging, it doesn't reload

**Reply by ndonkoHenri** [reply]
Which flet version are you using? And which OS?

**Reply by MichaelDecent** [reply]
Version 0.9.0 
Ubuntu Linux distro

**Reply by rkleivel** [reply]
@MichaelDecent  - check out this issue: https://github.com/flet-dev/flet/issues/1556
Does it seem like the same?

**Reply by MichaelDecent** [reply]
Yes, it is the same, 
I guess this is specific to Ubuntu

**Reply by rkleivel** [reply]
Ok - great that you also are able to reproduce it. If you have a look at the first video in that issue, you can see it works for some. But I have not been able to identify what makes the difference to my installation. 

Could you give a short description of your app environment setup?

**Reply by MichaelDecent** [reply]
I use VS code, 
I installed it using pip install method

**Reply by MichaelDecent** [reply]
were you able to find any solution

**Comment by rkleivel** [comment]
> were you able to find any solution

Unfortunately not yet. However, it is proven that it can work on ubuntu (see https://github.com/flet-dev/flet/issues/1556#issuecomment-1624887877) So it would be useful for us if @FeodorFitsner could share the setup and python environment of his Ubuntu installation (similarly to this video: https://github.com/flet-dev/flet/issues/1556#issuecomment-1630334583)

**Comment by Muddassir-Farooq-official** [comment]
Hot reload in flet is slow specially when there is more than one file in directory means each view in seperate file
